sirius/coding/test_image_coding.py

```diff
@@ -6,21 +6,29 @@
 
 class ImageCase(unittest.TestCase):
 
-    def _test_normal_text(self):
+    def test_normal_text(self):
         data = image_encoding.html_to_png('')
         image = Image.open(data)
         self.assertEquals(image.size[0], 384)
 
-    def _test_normal_height(self):
+    def test_normal_height(self):
         data = image_encoding.html_to_png(
             '<html><body style="margin: 0px; height: 100px;"></body></html>')
         image = Image.open(data)
+        self.assertEquals(image.size[0], 384)
         self.assertEquals(image.size[1], 100)
 
-    def _test_rle(self):
+    def test_probably_beyond_viewport_height(self):
```

Comment:
Trying to make sure that headless browsers can actually capture a full screenshot, and don't get limited to resizing to fit a virtual screen size (say, 1024x768)